app/viewmodels/RequiredDocumentsHelper.scala

```diff
@@ -60,31 +61,33 @@ object RequiredDocumentsHelper {
         case _ => None
       })
 
-  def formatNames(names: Seq[Name]): String = {
+  def formatNames(names: Seq[Name])(implicit messages: Messages): String = {
     val listOfNames = names.map(_.getFullName)
-    listOfNames.dropRight(1).mkString(", ") match {
-      case moreThanOneNames if moreThanOneNames.nonEmpty => moreThanOneNames + " and " + listOfNames.last
-      case _ => listOfNames.last
+    messages.lang.code match {
```

Comment:
could you please refactor this with reference to below code
https://github.com/hmrc/charities-registration-frontend/blob/master/app/controllers/operationsAndFunds/WhatCountryDoesTheCharityOperateInController.scala#L55